### ISSUE TITLE
When using the feature "Change height to fit.." the ScreenHeight() gets calculated wrong with retina displays

### ISSUE THREAD

**Issue by trsh** [4ian/GDevelop#1926]
## Describe the bug
When using the feature "Change height to fit screen or window" the ScreenHeight() gets calculated wrong with retina displays

## To Reproduce
1. Go to game settings > Properties > change "Game resolution resize mode" to  "Change height to fit screen or window"
2. Start network preview
3. Open the URL on mobile device **with Retina display**
4. Print out the ScreenHeight() to some text box
6. It prints height without retina taking into account (for me 720px)
7. If you then use this height for calculating stuff in your game (placing sprites, calc. boundaries, etc.) it's all wrong. 

Basically there is no much use for height number without retina taken into account. 

## Other details
* My mobile device has 2x retina (720px * 2 height), browser chrome, Android
* Gdevelop =Desktop versions 5.0.0-beta98

**Comment by trsh** [comment]
Is this confirmed?

**Comment by trsh** [comment]
I used `SceneWindowHeight` instead and it all just works fine.
